data-prep-pipeline/scripts/subtype_tree/create_subtype_tree.py

```diff
@@ -0,0 +1,618 @@
+import argparse
+import re
+import itertools
+import numpy as np
+import pandas as pd
+from taigapy import create_taiga_client_v3
+
+# TODO FOR 25Q2: REMOVE THE USE OF THIS TEMPORARY MODEL FILE
+temp_model_id = 'alison-test-649a.18/Model_temp_between_24q4_25q2'
```

Comment:
oh boy, this in a few places, huh? Definitely recommend asserts so we don't miss changing any